spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/TaskOperations.java

```diff
@@ -64,9 +64,10 @@
 	 * @param name the name of the task
 	 * @param properties the deployment properties
 	 * @param arguments the command line arguments
+	 * @param platformName the name of the platform to use
 	 * @return long containing the TaskExecutionId
 	 */
-	long launch(String name, Map<String, String> properties, List<String> arguments);
+	long launch(String name, Map<String, String> properties, List<String> arguments, String platformName);
```

Comment:
Another option, which is how it is done for skipper, is to put a special key in the properties map, just as for other deployer properties.